### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hollyallen**
Based on the 18F/Midas version of this file, this file should help folks know how to contribute to this repo and what to expect.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+This project is in the public domain within the United States, and
+copyright and related rights in the work worldwide are waived through
+the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+
+All contributions to this project will be released under the CC0
+dedication. By submitting a pull request, you are agreeing to comply
+with this waiver of copyright interest.
+
+## <a name="commit-messages"></a> Commit Messages
```
Comment (benbalter) [inline CONTRIBUTING.md]: Don't need the anchor here, AFAIK

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+paragraphs) describing the details of the commit.  Paragraphs are each
+separated by a blank line.  Paragraphs must be word wrapped to be no
+longer than 76 characters.
+
+The last part of the commit log should contain all "external
+references", such as which issues were fixed or URLs to external content.
+
+## Submitting a Pull Request
+
+We accept changes as Pull Requests. The [GitHub Help](https://help.github.com/articles/using-pull-requests/)
```
Comment (benbalter) [inline CONTRIBUTING.md]: "Accept" or "encourage you to propose"?

**Comment by hollyallen** [comment]
Good comments @benbalter , incorporated those changes.

**Comment by gbinal** [comment]
:+1: Thanks!

**Pull request merged**